lib/demos/todo.dart

```diff
@@ -197,3 +189,84 @@ class TodoAppState extends State<TodoApp> {
     scheduleUpdate();
   }
 }
+
+
+typedef void ChangeListener();
```

Comment:
Yes. Any signature conforms to `Function`. This specifically wants `void` and no args.